sofl/dialogs/install_dialog.py

```
@@ -430,189 +335,204 @@ def on_install_clicked(self, button):
-        
+
         try:
             game_id = self._generate_game_id()
-            new_game = self._create_game_object(game_id, game_name, install_path, executable)
-            
+            new_game = self._create_game_object(
+                game_id, game_name, install_path, executable
+            )
+
             # Add game to store
             shared.store.add_game(new_game, {}, run_pipeline=False)
```

Pylint (3.3.7)

[error] Module 'sofl.shared' has no 'store' member

(E1101)